src/components/ProtectedRoute.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import {
+  Route,
+  Redirect,
+  RouteComponentProps,
+  RouteProps,
+} from 'react-router-dom';
+import Auth from '../auth/auth';
+
+/* eslint-disable react/jsx-props-no-spreading */
```

Comment:
can this be removed? is this one you fixed globally?

Reply:
I think it wasn't added globally but if it's causing general issues then you can go ahead and add it to the ignore in global rules